src/public/services/AuthService.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+
+import { RedirectUrlDto } from 'src/types/api/auth'
+
+// Exported for testing
+export const PUBLIC_FORMS_ENDPOINT = '/api/v3/forms'
```

Comment:
I think it's fine to inline them since they are only used once

Reply:
i would prefer not to for `PUBLIC_FORMS_ENDPOINT` because for `validate`, it's going to be reused but i'll fix it for the rest